checklists/alz_checklist.en.json

```diff
@@ -2480,22 +2480,78 @@
         {
             "category": "Security",
             "subcategory": "Operations",
-            "text": "Connect default resource configurations to a centralized Azure Monitor Log Analytics workspace.",
+            "text": "Creating dedicated workspaces for Monitor and Secuity to to segregate ownership of data between operational and security teams and optimize costs.",
             "waf": "Security",
             "service": "Monitor",
             "guid": "e5f8d79f-2e87-4768-924c-516775c6ea95",
             "id": "G03.08",
-            "severity": "Medium",
+            "severity": "High",
             "link": "https://learn.microsoft.com/azure/azure-monitor/logs/design-logs-deployment"
         },
+        {
+            "category": "Security",
+            "subcategory": "Operations",
+            "text": "Implement SIEM solution to consolidate security data in a central location where it can be correlated across various services.",
+            "waf": "Security",
+            "service": "Entra",
+            "guid": "a56888b2-7e83-4404-bd31-b886528502d1",
+            "id": "G03.09",
+            "severity": "High",
+            "graph": "resources| where type == 'microsoft.operationalinsights/workspaces'| extend wsid = properties.customerId| project workspaceResourceId = tolower(id), name, wsid| join (resources| where type == 'microsoft.operationsmanagement/solutions'| where name has 'SecurityInsights'| extend workspaceResourceId = tostring(tolower(properties.workspaceResourceId))| project workspaceResourceId | summarize ResourceCount = count() by workspaceResourceId) on workspaceResourceId| extend RCount = iff(isnull(ResourceCount), 0, ResourceCount)| project-away ResourceCount| extend compliant = (RCount <> 0)",
+            "link": "https://learn.microsoft.com/en-us/azure/sentinel/connect-data-sources?tabs=azure-portal"
+        },
+        {
+            "category": "Security",
+            "subcategory": "Operations",
+            "text": "Facilitates proactive threat hunting and investigation via Sentinel - built-in threat detection mechanisms to detect and respond to security threats in real time.",
```

Comment:
This seems like it is not very verifiable or actionable - it is vague set of practices, and I'm not sure that the ALZ checklist is the right spot for this.  One of the other checklists might be better.